elements/pfe-styles/demo/index.html

```diff
@@ -132,5 +163,72 @@ <h4>Code</h4>
       </section>
     </article>
 
+    <article>
+      <header>
+        <h2>Color Palette</h2>
+      </header>
+      <section>
+        <section class="pfe-l-grid pfe-m-gutters pfe-m-all-4-col">
+          <button class="themes" on="light" name="light">Light theme</button>
+          <button class="themes" on="dark" name="dark">Dark theme</button>
+          <button class="themes" on="saturated" name="saturated">Saturated theme</button>
+        </section>
```

Comment:
Maybe add a basic text example too:

```suggestion
          <div class="container bg-light on-light">
            <p class="themes" name="light">Light theme</p>
          </div>
          <div class="container bg-dark on-dark">
            <p class="themes" name="dark">Dark theme</p>
          </div>
          <div class="container bg-saturated on-saturated">
            <p class="themes" name="saturated">Saturated theme</p>
          </div>
        </section>
```

Reply:
with this up in the CSS:
```css
      .bg-light {
        background-color: #fff;
        border: 1px solid #dedede;
      }

      .bg-dark {
        background-color: #252525;
      }

      .bg-saturated {
        background-color: #0477a4;
      }
```